docs/GVH7074.md

```diff
@@ -0,0 +1,64 @@
+# Notes for GVH7074
+
+https://github.com/w1gx/govee-ble-scanner/wiki/Sniffing-BLE-advertising-packets was helpful in decoding the GVH5074
+
+Full 'p' strings captured from ESP32 serial monitor
+```
+02010607030A18F5FE88EC1109476F7665655F48353037345F414141410AFF88EC00650775126402
+02010607030A18F5FE88EC1109476F7665655F48353037345F414141410AFF88EC00650769126402
+02010607030A18F5FE88EC1109476F7665655F48353037345F414141410AFF88EC0069075C126402
+```
+
+## Flags
+`02 01 06`
+
+`02` - two bytes 
+
+`01` - Type = Flags 
+
+`06` - Value (bitwise - 00000110) Bit 1 : "LE General Discoverable Mode", Bit 2: "BR/EDR Not Supported."
+
+## Service Class
+`07 03 0A 18 F5 FE 88 EC`
+
+`07` - 7 bytes 
+
+`03` - Type = Service Class 
+
+`0A 18` -> 18 0A (little endian) - Device Information 
+
+`F5 FE` -> FE F5 (little endian) - Dialog Semiconductor GmbH
+
+`88 EC` -> EC 88 (little endian) - unknown
+
+## Complete Local Name
+`11 09 47 6F 76 65 65 5F 48 35 30 37 34 5F 41 41 41 41`
+
+`11` - 17 bytes 
+
+`09` - Type = Complete Local Name 
+
+`47 6F 76 65 65 5F 48 35 30 37 34 5F 41 41 41 41` = Govee_H5074_AAAA (AAAA = last 4 of mac)
+
+## Manufacturer Specific Data
+`0A FF 88 EC 00 65 07 75 12 64 02`
+
+`0A` - 10 bytes 
+
+`FF` - Type = Manufacturer Specific Data 
+
+`88 EC 00` - unknown - same across all 'p' strings 
+
+`65 07` -> `07 65` (little endian) = 1893 in decimal. Divide by 100 to get temp in °C 
+
+`75 12` -> `12 75` (little endian) = 4725 in decimal. Divide by 100 to get relative humidity 
+
+`64` - 100 in decimal. Assuming this is battery percentage, but not sure. 
+
+`02` - unknown 
+
+## Temperature
```

Comment:
Can edit this section now if the `.geti` is working appropriately.

Reply:
The Two's complement is so close to what you figured out. It's subtracting the temp value from `0x10000` instead of from `0xFFFF`!